dapp/src/auctions/components/AuctionItem.tsx

```diff
@@ -120,41 +115,15 @@ export function AuctionItem({ auction, auctionStatus, onBidClick }: AuctionItemP
     };
 
     return (
-        <Card type={CardType.Filled}>
-            <div className="space-y-3">
-                <div className="flex items-center justify-between">
-                    <h3 className="font-medium text-lg">{auction.name}</h3>
+        <NameCard key={auction.name} name={auction.name}>
```

Comment:
Unnecessary isn't it?
```suggestion
        <NameCard name={auction.name}>
```